timesketch/lib/analyzers/sessionizer_test.py

```diff
@@ -231,6 +264,24 @@ def _create_mock_event(event_id, quantity, time_diffs=None):
 
         yield eventObj
 
+        # adding extra events after every requested event for better simulation
+        # of real timeline data i.e. working with a larger dataset
+        for _ in range(100):
+            ts += 1
+            event = event_template
```

Comment:
This block (up to L 280) is identical to the one in lines 254-263. Maybe extract to a method?